test/functional/page_objects/dashboard_page_controls.ts

```diff
@@ -375,11 +386,44 @@ export class DashboardPageControls extends FtrService {
     return +(await availableOptions.getAttribute('data-option-count'));
   }
 
-  public async optionsListPopoverGetAvailableOptions(filterOutExists: boolean = true) {
+  public async optionsListPopoverGetAvailableOptions() {
```

Comment:
Note that the `filterOutExists` parameter is no longer necessary because the tests that used this were removed as part of [removing the `"Allow exists"` toggle from the UI](https://github.com/elastic/kibana/pull/147216).